core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -308,29 +327,14 @@ var TopicsAndSkillsDashboardPage = function() {
       skillNameField,
       'Create Skill modal takes too long to appear.');
     await skillNameField.sendKeys(description);
-    await editConceptCardExplanationButton.click();
```

Comment:
It doesn't look like `editConceptCardExplanationButton` is used anywhere else, so can you delete that selector from the top?

Reply:
Done